### PR TITLE
typo: alertsmanagement/resource-manager/Microsoft.AlertsManagement

### PR DESCRIPTION
**Pull request by nschonni**
- Aler -> Alert
- Succesfully -> Successfully
- GetbyId -> GetById

### CODE REVIEW

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json [hunk 1]
```diff
@@ -176,7 +176,7 @@
         },
         "x-ms-examples": {
           "GetById": {
-            "$ref": "./examples/Alerts_GetbyId.json"
+            "$ref": "./examples/Alerts_GetById.json"
```
Comment (amarzavery) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json]: This file name is not changed.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json [hunk 2]
```diff
@@ -164,7 +164,7 @@
         },
         "x-ms-examples": {
           "GetById": {
-            "$ref": "./examples/Alerts_GetbyId.json"
+            "$ref": "./examples/Alerts_GetById.json"
```
Comment (amarzavery) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json]: This file name is not changed.

Comment (amarzavery) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json]: Actually do not change the file names. It creates unnecessary hassle to make sure that the file exists.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json (hunk 1 above)
Comment (nschonni) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json]: The renames are in the changes

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json (hunk 2 above)
Comment (amarzavery) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json]: Spell check fixes in description and readme are good. Let's not rename the files. What we have is good enough.
Please address this feedback to get the PR merged.

Comment (nschonni) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json]: I know it seems silly, but it's one of the things that gets flagged in #4836. If it's not fixed here, then it's going to need to be excluded later. There have already been a few PRs to fix these little casing things landed

Comment (nschonni) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json]: I did find that the modelValidation.js crashes hard when I did rename the reference without the file 😝

**Comment by nschonni** [comment]
@amarzavery I finally saw that the second rename was missing. Fixed and re-pushed

**Review by amarzavery — APPROVED** [review]

**Pull request merged**